### DISCUSSION TITLE
What is the usage of Flagd for feature flags?

### DISCUSSION THREAD

**Discussion by AlexPSplunk** [open-telemetry/opentelemetry-demo#1500] [Q&A]
The old featureflag service is out of the latest release, and it has been replaced with flagd.  There are some docs on flagd here, but it's pretty bare-bones.
What is the intended way of enabling or disabling a feature with flagd?  Do we manually edit the configmap "*-flagd-config"?
If we edit the configmap, do we need to restart the flagd pod?

**Comment by julianocosta89** [comment]
@AlexPSplunk, as of today you are correct, we need to manually update the flagd config.
In Docker the update is identified as we change the values in the file, but in k8s we need to restart the pod.

**Comment by puckpuck** [comment]
We now have flagd-ui which is used to set feature flags with a UI.
